prime-router/src/main/kotlin/cli/tests/Hl7Tests.kt.orig

```diff
@@ -1,223 +0,0 @@
-package gov.cdc.prime.router.cli.tests
```

Comment:
Why was this file removed? It's not a kotlin file, so it shouldn't be affected by the linter.

Reply:
same as the other .orig files